app/policies/audio_file_policy.rb

```diff
@@ -0,0 +1,5 @@
+class AudioFilePolicy < AccountablePolicy
+  def original?
+    update?
```

Comment:
Do we want to start actually checking scopes at this point? Might be a good place to start - in this case I'd say we can do something like `token.authorized?(resource.account, 'read-private')`